README.md

```diff
@@ -10,17 +10,35 @@ This GitHub Action (created using typescript) uses [CLI for Microsoft 365](https
 Create a workflow `.yml` file in your `.github/workflows` directory. An [example workflow](#example-workflow---cli-for-microsoft-365-login) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
 
 ### Inputs
-- `ADMIN_USERNAME` : **Required** Username (email address of the admin)
-- `ADMIN_PASSWORD` : **Required** Password of the admin
+- `ADMIN_USERNAME` : Username (email address of the admin)
+- `PASSWORD` : Password for the admin or the certificate
```

Comment:
If we rename `ADMIN_PASSWORD` to `PASSWORD`, wouldn't that be a breaking change?

Reply:
Sure yes. Would that be a problem?

Reply:
Unless necessary, we should avoid breaking changes as it causes extra work for our users to use the latest version. In this case, I'd suggest that we use two separate variables even if it means a bit more work for us.